src/OpenTelemetry.AutoInstrumentation/Plugins/PluginManager.ContinuousProfiler.cs.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Is there an issue with this file name, it has two `.cs'

Reply:
Fixed 37c7c5f50ba0593b6cdd63cf8a74549992ab0f50